src/backend/app/modules/Sharding/ApiResolvers/VodacomMobileMoneyApiResolver.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
I think we're gonna resolve companyId from JTW token in request instead from url segment. However, it's okay for draft.